data/hai/hai reveal 4 middle.txt

```diff
@@ -712,6 +714,7 @@ mission "Hai Reveal [A05] Take Teeneep to Hai-home"
 	to offer
 		has "event: alondo symbol timer"
 		or
+			has "ZZ Hai Reveal [A04] Report to Teeneep: done"
 			has "Hai Reveal [A04] Report to Teeneep: done"
```

Comment:
Duplication.
```suggestion
			has "ZZ Hai Reveal [A04] Report to Teeneep: done"
```

Reply:
That was to avoid breaking old saves during my testing.